index.html

```diff
@@ -189,6 +189,17 @@ <h2 id="setting-theme-heading" i18n-content="themeSetting"></h2>
                 </div>
               </div>
             </div>
+            <div class="group" role="radiogroup" aria-labelledby="setting-help-heading">
```

Comment:
Remove `role="radiogroup"`

Reply:
Whoops, good eye! Thanks for catching this.